### ISSUE TITLE
Should Advertise lobby bool

### ISSUE THREAD

**Issue by Linxmas** [betidestudio/EOSIntegrationKit#51]
Using find eik session with match type of lobby would still find lobby set with should advertise false.

**Comment by AlexWickedGames** [comment]
Yeah I just noticed tthis issue too. You actually can't make private lobbies in my game until this issue is fixed. I can't use private connections because private connections doesn't support joining the game by the session ID
